hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -92,9 +92,6 @@
 import static org.apache.hadoop.hdds.recon.ReconConfigKeys.OZONE_RECON_HTTP_ADDRESS_KEY;
 import static org.apache.hadoop.hdds.recon.ReconConfigKeys.OZONE_RECON_TASK_SAFEMODE_WAIT_THRESHOLD;
 import static org.apache.hadoop.hdds.scm.ScmConfig.ConfigStrings.HDDS_SCM_INIT_DEFAULT_LAYOUT_VERSION;
-import static org.apache.hadoop.ozone.MiniOzoneCluster.PortAllocator.anyHostWithFreePort;
-import static org.apache.hadoop.ozone.MiniOzoneCluster.PortAllocator.getFreePort;
-import static org.apache.hadoop.ozone.MiniOzoneCluster.PortAllocator.localhostWithFreePort;
```

Comment:
Please keep (and update) these static imports to minimize change elsewhere.

```suggestion
import static org.apache.ozone.test.GenericTestUtils.PortAllocator.anyHostWithFreePort
import static org.apache.ozone.test.GenericTestUtils.PortAllocator.getFreePort
import static org.apache.ozone.test.GenericTestUtils.PortAllocator.localhostWithFreePort
```

Reply:
Done!